.storybook/main.js

```diff
@@ -2,7 +2,10 @@ const custom = require("../webpack.config.js");
 
 module.exports = {
   // Where are the stories?
-  stories: ["../src/components/**/*.stories.tsx"],
+  stories: [
+    "../src/Intro.stories.mdx",
```

Comment:
Can we put this under a /stories or /docs folder so that we don't have to add to this whenever we have new standalone stories?